src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -94,7 +94,7 @@ void remove_redundant_reorders::run(program& p) {
             std::vector<program_node*> recalc_list;
 
             for (auto usr : node.get_users()) {
-                if (!lo.can_fuse_reorder(input, *usr, input.get_output_layout().format, usr->get_output_layout().format)) {
```

Comment:
nit: Maybe reduce the code length a bit by removing 5 `get_layout_optimizer()` function calls?
 And use `auto& lo = p.get_layout_optimizer()` in this pass too